perfkitbenchmarker/sample.py

```diff
@@ -33,10 +34,12 @@ def PercentileCalculator(numbers):
   count = len(numbers_sorted)
   total = sum(numbers_sorted)
   result = {}
-  for percentile in PERCENTILES_LIST:
+  for percentile in percentiles:
```

Comment:
Can you use numpy.percentile?
Any reason to write your own percentile calculator?